### PR TITLE
New design for Homepage

### PR DESCRIPTION
**Pull request by FRCassarino**
image

image

image

image

image

Remaining launch blocking tasks:
- @ingawei to add in extra design polish.
- Find and fix and new bugs/design issues introduced
- Fix the secrets management issue, so coverImgUrls start being stored, and use that instead of the random stock image.
- Daily mover profit/position should update when a quick bet happens.

Non launch blocking:
- Free Response markets don't yet match the figma 
- Show quickbetting arrows even when signed out, but they just redirect to the login dialog

### CODE REVIEW

File: web/components/contract/contract-card.tsx [hunk 1]
```diff
@@ -89,72 +91,42 @@ export function ContractCard(props: {
...
         />
-        <div
-          className={clsx(
-            'break-anywhere font-semibold text-indigo-700 group-hover:underline group-hover:decoration-indigo-400 group-hover:decoration-2',
-            questionClass
+        {showImage && (
+          <img
+            className="h-60 w-full object-cover "
+            src={
+              'https://firebasestorage.googleapis.com/v0/b/mantic-markets.appspot.com/o/dream%2FykU5KOeiSQ.png?alt=media&token=f7c69a0f-8f71-4758-996c-0b6411d16875'
```
Comment (FRCassarino) [inline web/components/contract/contract-card.tsx:104]: Will change this to contract.coverImgUrl before shipping - leaving for now so you can try the dev builds.

**Review by jahooma — APPROVED** [review]
Thank Fede, this is really great! A very impactful revamp to the homepage, card design, and quick bet component. 

And thanks, @ingawei, for the redesign! Glad to see you two working together.

It looks like this touches a lot of cases from open, closed, resolved markets to different market types. And you've done a great job handling them.

Here's two issues I found which should be fixed before merging:
- On mobile, the width overflows creating a horizontal scrollbar
Screen Shot 2022-10-26 at 3 51 17 PM
- Numeric markets show YES and NO in the quick bet bar. Should change to HIGHER and LOWER
Screen Shot 2022-10-26 at 3 54 30 PM

Otherwise, I think this is pretty much ready to go! I'd encourage merging ASAP as long as the most obvious bugs have been fixed.

The code looks pretty good too. Added Inga as a reviewer in case she spots anything. More work can be done to iterate after merging, of course.

**Comment by ingawei** [comment]
> 

Oh noo! I started making changes to this PR, just some UI things. @FRCassarino please give me at least today to do things lol

**Comment by FRCassarino** [comment]
Awesome work Inga, it looks much more polished now, and I see that you fixed the bugs James pointed out as well! And implemented  the FR market changes as well!

The only obvious bug that I see that's still around is this: 
>Daily mover profit/position should update when a quick bet happens.

I'll  fix that up and merge, unless there's any objections :) @jahooma @ingawei

**Pull request merged**